api/src/controllers/activity.ts

```diff
@@ -103,7 +104,7 @@ router.post(
 				data: record || null,
 			};
 		} catch (error: any) {
-			if (error instanceof ForbiddenException) {
+			if (isDirectusError(error) && error.code === ErrorCode.Forbidden) {
```

Comment:
There are frequent error code equality checks alongside `isDirectusError()`.

```ts
isDirectusError(error) && error.code === ErrorCode.Forbidden
```

Should the `isDirectusError()` be refactored to include an optional parameter for the error code equality check?

```ts
isDirectusError(error, ErrorCode.Forbidden)
```

Reply:
Love that idea! Good call